libc-test/build.rs

```diff
@@ -2091,6 +2095,7 @@ fn test_freebsd(target: &str) {
                 "sys/thr.h",
                 "sys/time.h",
                 [freebsd14 || freebsd15]:"sys/timerfd.h",
+                [freebsd14 || freebsd15]:"linux/input.h",
```

Comment:
This can be just `freebsd13 || freebsd14 || freebsd15` and then remove line 2072.